docs/manual/manual.adoc

```diff
@@ -451,6 +461,15 @@ is a CVE you can click it to display its metadata from the official NVD
 database (requires internet connection). OpenSCAP doesn't provide
 metadata for other types of identifiers.
 
+Another place, where these identifiers are located are machine-readable Result Datastream files.
```

Comment:
There is no need for a comma here as well, and maybe switch "are located" by "can be found":
Another place where these identifiers can be found is in the machine-readable Result Datastream files.